README.md

```diff
@@ -1,4 +1,44 @@
 # HelloBooksDef
 
 [comment]: # (HoundCi Badge)
-[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
\ No newline at end of file
+[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
+
+## Getting Started
+---
+### Installing
+
+To run this application, you need to have Node.js, and git(to clone the repo) installed. Then follow the instructions to get
+it up and running
+
+- clone the repo using 
+```shell
+~> git clone https://github.com/emp-daisy/HelloBooksLib.git
+```
+- run ``` npm install --prod``` to install dependencies
+- create a .env file from the .env.sample file and fill in the necessary environment variables
+- run ``` npm run build ``` to build the project and then run ``` npm start ``` to start the server
+- now access the server on the localhost port you specify e.g ``` localhost:5000 or 127.0.0.1:5000 ```
+
+Now the server will go live and listen for requests
+
+## Developing
+
+To develop the app further, a few handy tools have been put in place such as nodemon and some other dev dependencies.
+Access them by starting the server using ```npm run dev```. But before using the command, make sure to follow the steps below
+
+```shell
+git clone https://github.com/emp-daisy/HelloBooksLib.git
+cd banka/
```

Comment:
@chuxmykel this obviously isn't meant to be here right?

Reply:
@ocranbillions what exactly?

Reply:
Oh. Now I see it. I'll fix it. Thanks.